client/src/unifyfs-sysio.c

```diff
@@ -1600,59 +1464,335 @@ static int process_read_data(read_req_t* read_reqs, int count, int* done)
     return rc;
 }
 
+/* This uses information in the extent map for a file on the client to
+ * complete any read requests.  It only complets a request if it contains
+ * all of the data.  Otherwise the request is copied to the list of
+ * requests to be handled by the server. */
+static int service_local_reqs(
```

Comment:
It looks like this always returns `UNIFYFS_SUCCESS`, so make this a void function.

Reply:
...or alternatively return `out_count` and remove it as a function argument

Reply:
updated to return void